doc/changelog/08-tools/11255-master+fix11254-coqtop-version.rst

```diff
@@ -0,0 +1,4 @@
+- **Fixed:**
+  `coqtop --version` working even in the middle of an installation process
```

Comment:
Since there is the word "fixed" above, I'd rather formulate this negatively:

```suggestion
  `coqtop --version` not working in the middle of an installation process
```

Reply:
"`coqtop --version` was broken when called in the middle of an installation process" ??

Reply:
Even better.